### DISCUSSION TITLE
What's your branch protection strategy?

### DISCUSSION THREAD

**Discussion by AceHack** [meaningful-ooo/devcontainer-features#42] [Q&A] [answered]
I'm working on a feature repo and trying to set it up just like yours

**Comment by andreiborisov** [comment, marked as answer]
![Branch protection rule](https://github.com/meaningful-ooo/devcontainer-features/assets/13354491/ee7b803c-e9f4-43de-9f41-f0d8213dd75f)

We also use [a custom action](https://github.com/meaningful-ooo/devcontainer-features/blob/main/.github/workflows/fast-forward-pr.yaml) to merge via fast-forward since GitHub doesn't support this natively.
